Neos.ContentRepository.LegacyNodeMigration/Classes/Command/CrCommandController.php

```diff
@@ -164,6 +164,7 @@ private function adjustDataBaseConnection(Connection $connection): Connection
         $connectionParams['dbname'] = $this->output->ask(sprintf('DB name? [%s] ',$connectionParams['dbname'] ?? ''), $connectionParams['dbname'] ?? null);
         $connectionParams['user'] = $this->output->ask(sprintf('DB user? [%s] ',$connectionParams['user'] ?? ''), $connectionParams['user'] ?? null);
         $connectionParams['password'] = $this->output->askHiddenResponse(sprintf('DB password? [%s]', str_repeat('*', strlen($connectionParams['password'] ?? '')))) ?? $connectionParams['password'];
+        /** @phpstan-ignore-next-line  */
```

Comment:
I uhhh, we could try to fix everything to fit into the schema, but meh... See comment above.

Reply:
yes fine with ignoring ... but that didnt fix it yet ... ci is still red

Reply:
See above, I guess we have to ignore the password line too but I don't understand why, because the described schema has password. I get the same error locally, but then I do not undesrtand where the schema information comes from. (I guess because it's defined as psalm-param ?)